docs/source/operations/getting-started.rst

```diff
@@ -0,0 +1,354 @@
+
+Stopping
+========
+
+To stop the Substra stack, you need to stop the 3 components (backend, orchestrator and frontend) individually.
+
+* Stop the frontend: This action depends on which option you chose during the launch:
+
+  a. local server: Stop the process running the local server (usually using CONTROL + C)
+  b. Docker:
+
+     .. code-block:: bash
+
+      docker stop DOCKER_FRONTEND_CONTAINER_NAME
+
+     | with ``DOCKER_FRONTEND_CONTAINER_NAME`` the name of the frontend container you chose during the launch
+* Stop the orchestrator:
+
+  .. code-block:: bash
+
+   cd orchestrator
+   skaffold delete
+
+* Stop the backend:
+
+  .. code-block:: bash
+
+   cd substra-backend
+   skaffold delete
+
+If this command fails and you still have pods up, you can use the following command to remove the ``org-1`` and ``org-2`` namespaces entirely.
+
+.. code-block:: bash
+
+   kubectl rm ns org-1 org-2
+
+Next steps
+==========
+
+Now you are ready to go, you are ready to run either the :doc:`/auto_examples/index` or the :doc:`Substrafl (low-level library) examples </substrafl_doc/examples/index>` (low-level library).
+
+If you are interested in more deployment options or more customised set-up, you can have a look at :doc:`/operations/deploy` or at the documentation included in the repo of substra_, substra-backend_, orchestrator_ or substra-frontend_.
+
+Troubleshooting
+===============
+
+.. note::
+   Before going further in these section, you should check the following points:
+    * Check the version of Skaffold, Helm and Docker. For example, Skaffold is released very often and sometime it introduces bugs, creating unexpected errors.
+    * Check the version of the different Substra components:
+
+      * if you are using a release you can use :ref:`the compatibility table <additional/release:Compatibility table>`.
+      * if you are using the ``latest`` from main, check that you are up-to-date and see if there were any open issue in the repositories or any bugfixes in the latest commits.
+
+   You can also go through :doc:`the instructions one more time </operations/getting-started>`, maybe they changed since you last saw them.
+
+Troubleshooting prerequisites
+-----------------------------
+
+The errors in this category are linked with not reaching the hardware requirements. Please check if `you match these <#hardware>`__ first.
+
+* .. code-block:: pycon
+
+   <ERROR:substra.sdk.backends.remote.rest_client:Requests error status 502: <html>
+   <head><title>502 Bad Gateway</title></head>
+   <body>
+   <center><h1>502 Bad Gateway</h1></center>
+   <hr><center>nginx</center>
+   </body>
+   </html>
+
+   WARNING:root:Function _request failed: retrying in 1s>
+
+   You may have to increase the number of CPU for the backend in ``substra-backend/charts/substra-backend/values.yaml``
```

Comment:
I'm not sure what you're referring to here, is it the `resources.requests.cpu` key? However, we don't set those in our chart. Maybe you meant increase the number of CPU you assign to Docker for Mac?

(same below with RAM)

Reply:
I re-used part of the documentation available, My understanding indeed is that https://github.com/Substra/substra-backend/blob/c9181ce00d9093037e109507bcbcb3e2139d2f27/charts/substra-backend/values.yaml#L173-L182 only applies to deployment and we can use Docker limits for Docker on Mac

Reply:
Ok, so in this section we should probably suggest to increase the resources assigned to Docker for Mac, since changing the values won't help?